README.md

```diff
@@ -18,6 +18,17 @@ import "https://unpkg.com/dom-slot-assign@0.1"
 <script src="https://unpkg.com/dom-slot-assign@0.1"></script>
 ```
 
+If you're ok with [slightly reduced browser support](https://caniuse.com/mdn-javascript_operators_await_top_level),
+you can include it conditionally, only when needed:
+
+```js
+if (typeof HTMLSlotElement !== "undefined" && !HTMLSlotElement.prototype.assign) {
```

Comment:
Looking at [top level await support](https://caniuse.com/mdn-javascript_operators_await_top_level) and [optional chaining support](https://caniuse.com/mdn-javascript_operators_optional_chaining), I was wondering whether you think we should consider `globalThis?.HTMLSlotElement?.prototype.assign`?

I don’t have any opinion yet. Looking at your helpful addition just brought it to my mind. I’m including the change as a suggestion for your convenience, but feel free to ignore it.

```suggestion
if (globalThis?.HTMLSlotElement?.prototype.assign) {
```

Reply:
LOL. I need be careful coding anything today. 😂 Okay, let’s use it!